lib/PuppeteerSharp/Page.cs

```diff
@@ -14,18 +14,21 @@ namespace PuppeteerSharp
 {
     public class Page : IDisposable
     {
-        private Session _client;
-        private bool _ignoreHTTPSErrors;
-        private NetworkManager _networkManager;
-        private FrameManager _frameManager;
-        private TaskQueue _screenshotTaskQueue;
-        private EmulationManager _emulationManager;
+        private const int DefaultNavigationTimeout = 30000;
+
+        private readonly Session _client;
+        private readonly bool _ignoreHTTPSErrors;
+        private readonly NetworkManager _networkManager;
+        private readonly FrameManager _frameManager;
+        private readonly TaskQueue _screenshotTaskQueue;
+        private readonly EmulationManager _emulationManager;
+        
         private ViewPortOptions _viewport;
-        private Mouse _mouse;
         private Dictionary<string, Func<object>> _pageBindings;
-        private const int DefaultNavigationTimeout = 30000;
 
-        private static Dictionary<string, PaperFormat> _paperFormats = new Dictionary<string, PaperFormat> {
+        internal Mouse Mouse { get; }
```

Comment:
Let's make this public and move it to the public props section.
There are some tests that are going to use it.